internal/cni/cni.go

```diff
@@ -47,6 +47,7 @@ type NetConfig struct {
 	types.NetConf
 	Device   string `json:"deviceID"`
 	Mode     string `json:"mode"`
+	UnloadBpf bool   `json:"unloadBpf,omitempty"`
```

Comment:
NIT: code is out of line with surrounding code
Think a `make format` should line things up